docs/management/admin/response-actions-config.asciidoc

```diff
@@ -6,8 +6,68 @@
 :frontmatter-tags-content-type: [how-to]
 :frontmatter-tags-user-goals: [manage]
 
+Endpoint response actions involving third-party systems require additional configuration. This page explains the high-level steps you'll need to take to enable these response actions.
+
+[discrete]
+[[configure-sentinelone-response-actions]]
+== Configure SentinelOne response actions
+
+SentinelOne response actions allow you to perform bidirectional actions on protected hosts, such as directing SentinelOne to isolate a suspicious endpoint from your network, without needing to leave the {elastic-sec} UI.
+
+preview::[]
+
+.Prerequisites
 [sidebar]
 --
-[.text-center]
-**This is a placeholder for future documentation.**
+* https://www.elastic.co/pricing[Subscription level]: Enterprise 
+
+* {kibana-ref}/kibana-role-management.html#adding_kibana_privileges[{kib} feature privilege]: **Actions and Connectors : All**.
+
+* <<endpoint-management-req,{elastic-sec} feature privileges>>: **All** for any response action features, such as **Host Isolation**, that you want to perform.
+
+* Endpoints must have actively running SentinelOne agents installed.
 --
+
+Configuration requires the following general steps. Expand the steps and follow the links for detailed instructions:
+
+. **Generate an API access token in SentinelOne.** You'll need this token in later steps, and it allows {elastic-sec} to collect data and perform actions in SentinelOne. You must create the token with the SentinelOne permissions required for the actions you want to perform (for example, isolating and releasing an endpoint).
++
+Refer to the {integrations-docs}/sentinel_one[SentinelOne integration docs] or SentinelOne's docs for details on generating an API token.
+
+. **Install the SentinelOne integration.** The {integrations-docs}/sentinel_one[SentinelOne integration] uses the SentinelOne API to collect and ingest logs into {elastic-sec}.
++
+.Expand for details
+[%collapsible]
+====
+.. In {kib}, go to **Integrations**, search for and select **SentinelOne**, then select **Add SentinelOne**.
+.. Configure the integration with an **Integration name** and optional **Description**.
+.. Ensure that **Collect SentinelOne logs via API** is selected, and enter the required **Settings**:
+   - **URL**: The SentinelOne console URL.
+   - **API Token**: The SentinelOne API access token you generated previously.
+.. Scroll down and enter a name for the agent policy in **New agent policy name**. If other agent policies already exist, you can click the **Existing hosts** tab and select an existing policy instead. For more details on {agent} configuration settings, refer to {fleet-guide}/agent-policy.html[{agent} policies].
+.. Click **Save and continue**.
+.. If you need to install {agent} on any SentinelOne-protected hosts, select *Add {agent} to your hosts* and continue with the <<enroll-agent,{agent} installation steps>>. 
```

Comment:
Do we already document "agent-less" fleet policies somewhere? 

if not, then I would suggest clarifying here that this integration/fleet agent should be deployed to 1 agent only. That agent will act as the "middle" process in pulling the data out of the SentinelOne management platform and then sending that to elasticsearch, thus it does not need to be deployed to more than 1 host (and likely would never be deployed to a host running the SentinelOne agent)

Reply:
Agent-less policies is not something (I think) that exists yet, but something that has been talked about. The current work-around is to instruct the customer to deploy this fleet policy to a single host using Elastic Defenc (for the reason I highlighted above).

I guess the edit would need to happen on the line with: `...on any SentinelOne-protected hosts...` ⬅️ since this is not really true.

Reply:
Revised the Agent installation step in https://github.com/elastic/security-docs/pull/4593/commits/ac6b1443f12e18e446d37799406f76d59d8fa99a